apps/agreement/contracts/Agreement.sol

```diff
@@ -924,7 +922,7 @@ contract Agreement is IAgreement, AragonApp {
     */
     function _unlockAndSlashBalance(Staking _staking, address _user, uint256 _unlockAmount, address _challenger, uint256 _slashAmount) internal {
         if (_unlockAmount != 0 && _slashAmount != 0) {
-            _staking.unlockAndSlash(_user, _unlockAmount, _challenger, _slashAmount);
+            _staking.slashAndUnlock(_user, _challenger, _unlockAmount, _slashAmount);
```

Comment:
I thought we agreed on using `slashAndUnstake` here for now.
The UI is not thought to have the challenger interacting with the staking for now, we will probably do it in the future tho.